build.gradle.kts

```diff
@@ -63,21 +65,30 @@ tasks.withType<Test> {
 detekt {
     config.setFrom("$projectDir/config/detekt.yml")
     autoCorrect = true
+    source.setFrom(
+        files(
+            "$projectDir/src/main/kotlin",
+            "$projectDir/src/test/kotlin",
+            "$projectDir/src/testFixtures/kotlin",
+            "$projectDir/build.gradle.kts",
+        ),
+    )
```

Comment:
1) This is hardcode, if any new sourceSet will be added you might need to update this list
2) You don't need lint `build.gradle.kts`, it has a specific syntax.

I suggest
```kts
    source.setFrom(sourceSets)
```

Reply:
build.gradle.kts has gis own syntax but its fine to have trailing commas, spaces, indents etc in the right way
you suggestion with source sets does not check testFixtures niether